tests/test_tools.py

```diff
@@ -10,15 +10,41 @@
 from xonsh.platform import ON_WINDOWS
 from xonsh.lexer import Lexer
 from xonsh.tools import (
-    subproc_toks, subexpr_from_unbalanced, is_int, always_true, always_false,
-    ensure_string, is_env_path, str_to_env_path, env_path_to_str,
-    escape_windows_cmd_string, is_bool, to_bool, bool_to_str,
-    is_bool_or_int, to_bool_or_int, bool_or_int_to_str,
-    ensure_int_or_slice, is_float, is_string, is_callable,
-    is_string_or_callable, check_for_partial_string, CommandsCache,
-    is_dynamic_cwd_width, to_dynamic_cwd_tuple, dynamic_cwd_tuple_to_str,
-    is_logfile_opt, to_logfile_opt, logfile_opt_to_str, argvquote,
-    executables_in, find_next_break, expand_case_matching)
+    CommandsCache,
+    always_false,
+    always_true,
+    argvquote,
+    bool_or_int_to_str,
+    bool_to_str,
+    check_for_partial_string,
+    dynamic_cwd_tuple_to_str,
+    ensure_int_or_slice,
+    ensure_string,
+    env_path_to_str,
+    escape_windows_cmd_string,
+    executables_in,
+    expand_case_matching,
+    find_next_break,
+    is_bool,
+    is_bool_or_int,
+    is_callable,
+    is_dynamic_cwd_width,
+    is_env_path,
+    is_float,
+    is_int,
+    is_int_as_str,
+    is_logfile_opt,
+    is_slice_as_str,
+    is_string,
+    is_string_or_callable,
+    logfile_opt_to_str,
+    str_to_env_path,
+    subexpr_from_unbalanced,
+    subproc_toks,
+    to_bool,
+    to_bool_or_int,
+    to_dynamic_cwd_tuple,
+    to_logfile_opt)
```

Comment:
For the record autopep8 made that change. After working with it for a while, I don't disagree with it. 

My whole point in doing this was because in the _unalphabetized wall of text_ it was not obvious to me which functions were being imported from tools.

With each import is on its own line and in alphabetical order then it is easy to determine what has been imported. It is also easy to keep it alphabetized. These seem like worthwhile trade-offs, and pep8 compliant to boot.

I guess this doesn't make sense to me: "Vertical whitespace is precious". This doesn't seem like a Pythonic statement. Pep8 recommends using vertical whitespace in many places.